packages/thirdweb/src/wallets/in-app/native/native-connector.ts

```diff
@@ -110,15 +110,13 @@ export class InAppNativeConnector implements InAppConnector {
       case "google":
       case "facebook":
       case "apple": {
-        if (!params.redirectUrl) {
-          throw new Error(
-            "redirectUrl deeplink is required for oauth login (ex: myApp://) - You also need add this deeplink in your allowed `Redirect URIs` under your API Key in your thirdweb dashboard: https://thirdweb.com/dashboard/settings/api-keys",
-          );
-        }
+        const ExpoLinking = require("expo-linking");
+        const redirectUrl =
+          params.redirectUrl || (ExpoLinking.createURL("") as string);
```

Comment:
An empty url works here?

Reply:
Yeah, the string param is for the path after the base url